charts/incubator/splunk/Chart.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: splunk
+version: 8.1.5
+appVersion: auto
+description: Splunk for TrueNAS SCALE
```

Try leaving out the 'for TrueNAS SCALE' it's a bit redundant.

```suggestion
description: Splunk Enterprise is a platform for operational intelligence
```